src/ol/webgl/TileTexture.js

```diff
@@ -84,6 +93,7 @@ function uploadDataTexture(helper, texture, data, size, bandCount) {
     textureType,
     data
   );
+  gl.pixelStorei(gl.UNPACK_ALIGNMENT, 4);
```

Comment:
We can restore this to the previous `gl.getParameter(gl.UNPACK_ALIGNMENT)` value.  I've changed this in the latest commit.